README.md

```diff
@@ -15,7 +15,7 @@ trading CPU time for improved compression ratio.
 All versions feature the same decompression speed.
 
 LZ4 is also compatible with [dictionary compression](https://github.com/facebook/zstd#the-case-for-small-data-compression),
-both at [API](https://github.com/lz4/lz4/blob/v1.8.3/lib/lz4frame.h#L481) and [CLI](https://github.com/lz4/lz4/blob/v1.8.3/programs/lz4.1.md#operation-modifiers) levels.
+both at Aplication Program Interface [API](https://github.com/lz4/lz4/blob/v1.8.3/lib/lz4frame.h#L481) and Command Line Input [CLI](https://github.com/lz4/lz4/blob/v1.8.3/programs/lz4.1.md#operation-modifiers) levels.
```

Comment:
nit: `Application` takes 2 `p`.

Also note that the generally accepted expansion of API is "Application _Programming_ Interface" : https://en.wikipedia.org/wiki/Application_programming_interface

Finally, `CLI` is generally translated as "Command Line _Interface_" : https://en.wikipedia.org/wiki/Command-line_interface